File: CounterStrikeDashboard.Core/Model/Map.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CounterStrikeDashboard.Core.Model
{
    public class Map
    {
        public Map()
        {
            Players = new List<PlayerTuple>();
            Teams = new List<TeamTuple>()
            {
                new TeamTuple() { Team = "CT" },
                new TeamTuple() { Team = "T" },
            };
        }

        public DateTime Start { get; set; }
        public string MapName { get; set; }
        public List<PlayerTuple> Players { get; set; }
        public List<TeamTuple> Teams { get; set; }
        public bool Active { get; set; }
    }
}

File: CounterStrikeDashboard/Client/App/Models/Map.js
var csdash;
(function (csdash) {
    var Map = (function () {
        function Map() {
            this.__name__ = "Map";
            this.start = new Date();
            this.mapName = "";
            this.players = [];
            this.teams = [];
            this.active = false;
        }
        return Map;
    })();
    csdash.Map = Map;
})(csdash || (csdash = {}));
